### ISSUE TITLE
Lozi - Devops Engineer - Saigon - FT

### ISSUE THREAD

**Issue by thinhx** [awesome-jobs/vietnam#705]
<img src="https://user-images.githubusercontent.com/32698218/31434261-10ecb474-aea6-11e7-99fb-f8ca31a4690d.png" width="169" />

*If you may ask, Lozi is now one of the most lovable selling-community in Vietnam. Lozi's mission is to lead you a happy life by selling out second hand stuffs. We are built from scratch and now become the top-notch in Vietnam mobile market.*

*Millions of people using Lozi. This figure is growing up rapidly day by day. That's why we need more people to help us write our next chapter. Join a passionate, driven, aspiring team working to make the best experiences for sellers and sharing individual lifestyle. Dip the finger, test the waters, find your gig here at Lozi. Good chance comes only once ;)*

*The Lozi Story: A, B, C, X, Z, Y*
## Location

* Saigon, Vietnam

## Salary Expectation

* Negotiable

### Typical Daily Tasks

* Maintenance of servers and services.
* Active involvement in design, implementation, and maintenance of the development, staging, and production infrastructure and services.
* Work on automating tasks using Gitlab-ci/Jenkins.
* Troubleshoot system issues (such as high-load, memory, CPU usage, etc.) and come up with temporary/long term solutions based on the root cause.
* Work with developers to deploy applications ready for production (NGINX, Haproxy).
* Write scripts to automate configuration management (Fabric/Ansible)
* Setup deployment, backup, and restore strategies for PostgreSQL, Elasticsearch, Redis and other data-stores.
* Ensure proper security, monitoring, alerting and reporting for the infrastructure.
* Hands on maintenance on our Node.js, and Golang, Java applications.
* Collaborate closely with developers to make sound infrastructure decisions and improve workflow.
* Troubleshoot issues across the whole stack: hardware, software, and network.
* Document current and future procedures, configuration and policies.
* Monitor and plan for capacity upgrades.

### Skills & Requirements

* 2+ years of solid Linux/UNIX systems engineer/administrator experience.
* Automation experience with configuration management tools such as Fabric, Ansible.
* Proficient in bash shell scripting (sed + awk) and one of Ruby or Python.
* Experience in running apps built in Golang, Node.js.
* Experience in deploying, configuring, and maintaining NGINX, HAProxy, PostgreSQL, Elasticsearch, Fluentd, Redis, RabbitMQ 
* Experience automating application deployments with Fabric, Jenkins or another tooling.
* Ability to work in a proactive manner and manage your own queue.
* Experience with VMware vSphere or Security/Pen Testing a plus.
* Hands on experience with Docker and Docker Swarm/Kubernetes.

### Nice to have

- Good background with the Go programming language and its ecosystem 
- Basic web development knowledge(  HTML, CSS, Javascript/Node.Js )
- Good working knowledge of data structures, algorithms, Object Oriented design, Cloud, web sockets, microservices

### Contact

If you are interested, kindly send your CV to us with the subject [Lozi] - Backend Developer - [Your name]

* **Address:** Lozi 4F, Mirae Building (Big C), 268 To Hien Thanh str, District 10, HCMC
* **Email:** hola@lozi.vn or minhtriet@lozi.vn
* **Link:** https://lozi.vn/career/65
